DOCS/man/mpv.rst

```diff
@@ -371,6 +371,79 @@ To use this feature, you need to fill the ``menu-data`` property with menu
 definition data, and add a keybinding to run the ``context-menu`` command,
 which can be done with a user script.
 
+Image Bindings
+--------------
+
+Several key bindings are changed when viewing images:
+
+LEFT, RIGHT, UP, DOWN, h, j, k, l
+    Pan the image when it is larger than the window.
+
+Ctrl+LEFT, Ctrl+RIGHT, Ctrl+UP, Ctrl+DOWN, Ctrl+h, Ctrl+j, Ctrl+k, Ctrl+l
+    Align the image to one of the boundaries of the window when it is larger
+    than the window.
+
+= and -
+    Change the zoom.
+
++ and _
+    Change the zoom slowly.
+
+0
+    Reset the zoom.
+
+u
+    Toggle between showing the image unscaled in its original dimensions and
+    fitting it to the window.
+
+o
+    Fill the window with the image.
+
+s
+    Toggle between showing images for 5 seconds in a slideshow and keeping them
+    open forever.
+
+n
+    Go to the next playlist entry.
+
+p
+    Go to the previous playlist entry.
+
+N
+    Go to the next sub-playlist (e.g. directory or achive).
+
+P
+    Go to the previous sub-playlist.
+
+g-g
+    Go to the first playlist entry.
+
+G
+    Go to the last playlist entry.
+
+]
+    Skip 10 playlist entries forwards.
+
+[
+    Skip 10 playlist entries backwards.
+
+r and R
+    Rotate counterclockwise.
+
+t
+    Rotate clockwise.
+
+v
+    Rotate by 180 degrees.
```

Comment:
Add `m` for mirror?

Reply:
Do you mean flip or vflip? By the way, these are sxiv's keybindings if you want to copy the flip bindings (never needed flipping personally):

< Rotate image counter-clockwise by 90 degrees.
> Rotate image clockwise by 90 degrees.
? Rotate image by 180 degrees.
| Flip image horizontally.
_ Flip image vertically.

In sxiv r reloads the image, but I don't find that useful, so I bind r to rotate instead so you don't hold shift.